common/flatpak-run.c

```diff
@@ -2068,6 +2070,20 @@ flatpak_run_add_pulseaudio_args (GPtrArray *argv_array,
 }
 
 static void
+flatpak_run_add_pipewire_args (GPtrArray *argv_array,
+                                 char    ***envp_p)
+{
+
+  g_autofree char *pipewire_socket = g_build_filename (g_get_user_runtime_dir (), "pipewire-0", NULL);
+  if (g_file_test (pipewire_socket, G_FILE_TEST_EXISTS))
+    {
+      add_args (argv_array,
+                "--bind", pipewire_socket, pipewire_socket,
```

Comment:
This is not right. We always set the user runtime dir to /run/user/$uid in the sandbox, but it can be elsewhere on the outside, which is what g_get_user_runtime_dir() reports, so we need two different args here. That is why e.g. the wayland code does:

```
  wayland_socket = g_build_filename (g_get_user_runtime_dir (), wayland_display, NULL);
  sandbox_wayland_socket = g_strdup_printf ("/run/user/%d/%s", getuid (), wayland_display);
```